frontend/Jenkinsfile

```diff
@@ -5,40 +5,33 @@
 pipeline {
   agent any
   environment {
-    OUTFILE = "s3://ccdscan.concordium.com/${TARGET_NET}/ccdscan-frontend-${VERSION}.tar.gz"
+    image_repo = 'concordium/ccdscan-frontend'
+    image_name = "${image_repo}:${image_tag}"
   }
   stages {
-    stage('build') {
-      agent {
-        docker {
-          image 'node:16'
-          // Run the container on the node specified at the
-          // top-level of the Pipeline, in the same workspace,
-          // rather than on a new node entirely:
-          reuseNode true
-        }
+    stage('dockerhub-login') {
+      environment {
+        CRED = credentials('jenkins-dockerhub')
+      }
+      steps {
+        sh 'echo $CRED_PSW | docker login --username $CRED_USR --password-stdin'
       }
+    }
+    stage('build') {
       steps {
         sh '''\
-          cd frontend
-          
-          # Install dependencies
-          yarn install
-
-          # Run linting
-          yarn lint
-
-          # Build application bundle
-          env ENVIRONMENT="${TARGET_NET}" yarn build
-
-          # Package
-          tar cvzf out.tar.gz firebase.json .output
+          docker build \
+            --label git_commit="$GIT_COMMIT" \
+            --build-arg ENVIRONMENT=${TARGET_NET} \
```

Comment:
Future request in a different PR. 

Could we make a build for all the different environments, and let it be decided upon runtime which one we use instead? So we have a single docker image.

Reply:
As per discussed in person we will keep it as is.